src/components/editor/markdown-renderer/markdown-it-plugins/task-lists.ts

```diff
@@ -0,0 +1,143 @@
+// Markdown-it plugin to render GitHub-style task lists; see
+//
+// https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments
+// https://github.com/blog/1825-task-lists-in-all-markdown-documents
+
+import MarkdownIt from 'markdown-it/lib'
+import StateCore from 'markdown-it/lib/rules_core/state_core'
+import Token from 'markdown-it/lib/token'
+
+let disableCheckboxes = true
+let useLabelWrapper = false
+let useLabelAfter = false
+
+interface TaskListsOptions {
+  enabled: boolean
+  label: boolean
+  labelAfter: boolean
+}
+
+type TokenConstructorType = typeof Token
+
+export default function markdownItTaskLists (md: MarkdownIt, options: TaskListsOptions): void {
```

Comment:
We try to avoid default exports, because we prefer to be explicit with the imports.